importexport/musicxml/importmxmlpass1.cpp

```diff
@@ -1101,8 +1101,11 @@ void MusicXMLParserPass1::identification()
             else if (_e.name() == "encoding") {
                   // TODO
                   while (_e.readNextStartElement()) {
-                        if (_e.name() == "supports" && _e.attributes().value("element") == "beam" && _e.attributes().value("type") == "yes")
-                              _hasBeamingInfo = true;
+                        if (_e.name() == "supports" )
+                              if (_e.attributes().value("element") == "beam" && _e.attributes().value("type") == "yes")
+                                    _hasBeamingInfo = true;
+                              else if (_e.attributes().value("element") == "transpose")
```

Comment:
This results in a compiler warning:
`warning: suggest explicit braces to avoid ambiguous 'else' [-Wdangling-else]`

Reply:
Will do, thanks!

Reply:
#8481